README.md

```diff
@@ -25,6 +25,13 @@ To run it together with an `IEx` console you can do:
 iex -S mix dev
 ```
 
+If this is the first time that you launch the dev script you may need to generate the configuration files and adapt their content to your local environment:
+
+```
+cp config/dev.example.exs config/dev.exs
+cp config/test.example.exs config/test.exs
```

Comment:
It would be nice to reference also the usage of `mix deps.get` and `mix setup`

Reply:
Yeah. I made a few extra modifications to the README. I've also removed the `mix setup` script as it doesn't work since we are not using `npm` anymore.

The current instructions guide you to use `mix do assets.install, assets.build` which is much better.